pkgs/development/tools/misc/dura/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib, fetchFromGitHub, rustPlatform, pkgs }:
```

Comment:
The use of `pkgs` is discouraged. Please explicitly pass the packages you need.

Reply:
Fixed, thanks !